docs/development/pigs/pig-017.rst

```diff
@@ -0,0 +1,217 @@
+.. include:: ../../references.txt
+
+.. _pig-017:
+
+***************************
+PIG 17 - Provenance capture
+***************************
+
+* Author: José Enrique Ruiz, Mathieu Servillat, lead developers welcome...
```

Comment:
Suggest to remove "lead developers welcome" now and then to add anyone that makes significant contributions to this PIG (usually via comments, not edits) as co-authors.